src/ert/scheduler/lsf_driver.py

```diff
@@ -102,6 +103,39 @@ def parse_bjobs(bjobs_output: str) -> Dict[str, Dict[str, Dict[str, str]]]:
     return {"jobs": data}
 
 
+def build_resource_requirement_string(
+    exclude_hosts: Sequence[str], resource_requirement: Optional[str]
+) -> str:
+    # Create a string representing the exclusion of hosts if any are provided.
```

Comment:
Since this is the first comment, it can be interpreted as the functions docstring. Maybe move it to inside the "if"-statement